### DISCUSSION TITLE
Tutorials for beginners?

### DISCUSSION THREAD

**Discussion by francispoulin** [CliMA/Oceananigans.jl#2473] [General]
This summer I have 3 undergraduate researchers who will be learning to use Oceananigans.  I am planning to get them to read the docs, but wondered whether anyone might have written a tutorial that is a more condensed versoin of the docs to give people hands on experience?  

I am happy to put something together and share it, if there's interest, but thought I would check in case someone else has already done this.  Also, might be helpful for JuliaCon?

**Comment by glwagner** [comment]
I think the examples are meant to serve as a sort of tutorial (though as anything its a work in progress). We usually recommend that people go through the examples to get started. What kind of tutorial did you have in mind?

**Reply by francispoulin** [reply]
Good question! 

My vision was to have a slide or notebook prepared that would introduce Oceananigans to the new user, give them an overview of what's there, but also give them some time to run code and try stuff out.  Since nothing exists yet, I'll put something together and post it here when I think it's useful/interesting.  

I will certainly be borrowing from the docs, but the key would be to condense it into less than an  hour.

**Comment by glwagner** [comment]
I'm also hoping to add this Makie tutorial to the docs: https://github.com/CliMA/ClimaWorkshops/blob/main/intro-makie/barotropic_turbulence.jl

but haven't found the time yet.

**Reply by francispoulin** [reply]
I need to learn Makie so this is a great link, thanks!  I'll take a look and see what I can figure out.
